### PR TITLE
Fixes #191. Add Configuration doesn't show PHP

### LINKED ISSUE #191: "Add Configuration..." not displaying PHP as an option
I'm not able to add a config for a PHP debug session as it's not showing up in the add config menu.

I'm on 1.16.0 and have php 7.1.5 installed.

It might also be able to put a complete config in the docs with some indication of what is required and what is optional.

### PR DESCRIPTION
**Pull request by blockscoped**
When selecting `Add Configuration` in a VS Code launch.json file PHP is no longer shown as an option. The new `configurationSnippets` resolves this.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -188,20 +188,28 @@
             }
           }
         },
-        "initialConfigurations": [
```
Comment (felixfbecker) [inline package.json]: Why did you remove `initialConfigurations`?

Comment (tom-shan) [reply package.json:201]: we can reserve `initialConfigurations` and add `configurationSnippets` like the third suggestion in https://github.com/felixfbecker/vscode-php-debug/issues/318.

Comment (tom-shan) [reply package.json:201]: fwiw
```
        "configurationSnippets": [
          {
            "label": "PHP: Listen for XDebug",
            "description": "Listening for XDebug",
            "body": {
              "name": "Listen for XDebug",
              "type": "php",
              "request": "launch",
              "port": 9000
            }
          },
          {
            "label": "PHP: Launch currently open script",
            "description": "Launch currently open script",
            "body": {
              "name": "Launch currently open script",
              "type": "php",
              "request": "launch",
              "program": "^\"${1:\\${file\\}}\"",
              "cwd": "^\"${2:\\${fileDirname\\}}\"",
              "port": 9000
            }
          }
        ]
```

**Comment by zobo** [comment]
I tried to apply this PR to main, but can’t seem to get it working. The snippets that show up in launch.json do not include these defined here. I tried to build and install the extension locally. The structure also seem similar to other adapters of the same kind. Any ideas?

**Comment by zobo** [comment]
Due to larger refactors of the codebase I decided to reimplement the fix with additional changes. Moved the code to #527.
